packages/mobile/src/components/offline-downloads/OfflineDownloader.tsx

```diff
@@ -1,11 +1,14 @@
 import { useEffect, useState } from 'react'
 
 import { useIsOfflineModeEnabled } from 'app/hooks/useIsOfflineModeEnabled'
+import { useLoadOfflineTracks } from 'app/hooks/useLoadOfflineTracks'
+import { DOWNLOAD_REASON_FAVORITES } from 'app/services/offline-downloader'
 import { startDownloadWorker } from 'app/services/offline-downloader/offline-download-queue'
 
 export const OfflineDownloader = () => {
   const isOfflineModeEnabled = useIsOfflineModeEnabled()
   const [initialized, setInitialized] = useState(false)
+  useLoadOfflineTracks(DOWNLOAD_REASON_FAVORITES)
```

Comment:
Question here: useLoadOfflineTracks requires a collection, but not sure if this should be necessary. Can I update the hook to not require this prop/should a different collection be passed here?

Reply:
Great point. https://github.com/AudiusProject/audius-client/pull/2376 is merged and has the fix, so you should be able to merge that in and remove the argument now.

Reply:
fixed, thanks 👍